src/test/groovy/graphql/archunit/NullabilityAnnotationUsageTest.groovy

```diff
@@ -1,4 +1,4 @@
-package graphql
+package graphql.archunit
 
```

Comment:
I'm going to write a few more ArchUnit tests, let's start organising them in 1 place